servant-foreign/test/Servant/ForeignSpec.hs

```diff
@@ -6,9 +6,11 @@ module Servant.ForeignSpec where
 import Data.Monoid ((<>))
 import Data.Proxy
 import Servant.Foreign
+import Servant.API.Internal.Test.ComprehensiveAPI
```

Comment:
I could also add them in my basicAuth PR.  the relevant issue is #465   whatever you want.  You can merge this without that change